RestImageResize/Security/QueryAuthorizer.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace RestImageResize.Security
+{
+    /// <summary>
+    /// Authorizes image transformation queries.
+    /// </summary>
+    public class QueryAuthorizer
```

Comment:
would be really nice to create IQueryAuthorized contract so user can override authorization using DI